src/StoryForm.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import request from 'superagent'
+
+class StoryForm extends Component {
+
+    constructor(){
+      super()
+      this.state = {"voteTitle": "", "voteUrl": "", "voteTotal": 0}
+      this.handleTitleChange = this.handleTitleChange.bind(this)
+      this.handleUrlChange = this.handleUrlChange.bind(this)
+      this.handleSubmit = this.handleSubmit.bind(this)
+    }
+
+    handleSubmit(e){
+      e.preventDefault()
+      this.createStory()
+    }
+
+    handleTitleChange(e){
+      this.setState({"voteTitle": e.target.value})
+    }
+
+    handleUrlChange(e){
+      this.setState({"voteUrl": e.target.value})
+    }
+
+    createStory(){
+      var that = this
+      request.post('http://master.bass-seahorse-cod.app.push.drieapp.co/stories/')
+      .set('Content-Type', 'application/json')
+      .send({name: that.state.voteTitle, url: that.state.voteUrl, size: that.state.voteTotal})
```

Comment:
I'm not sure entirely what is meant here. Do you mean mapping the keys to the name? So name: that.state.name, url: that.state.url, size: that.state.size ?